kong/cmd/restart.lua

```diff
@@ -10,6 +10,7 @@ local function execute(args)
   end
 
   pcall(stop.execute, args)
+  ngx.sleep(0.1)
```

Comment:
Now properly implemented with a `kill -0` loop to make sure Nginx is properly terminated before trying to start it again.